lib/miq_expression.rb

```diff
@@ -524,7 +524,22 @@ def self._to_ruby(exp, context_type, tz)
       operands = operands2rubyvalue(operator, exp[operator], context_type)
       clause = operands.join(" #{normalize_ruby_operator(operator)} ")
     when "contains"
-      operands = operands2rubyvalue(operator, exp[operator], context_type)
+      exp[operator]["tag"] ||= exp[operator]["field"]
+      operands = if context_type != "hash"
```

Comment:
:+1: yea. I put part 2 into my response because I know you often like to work that way.

LGTM